apt_offline_core/AptOfflineCoreLib.py

```diff
@@ -1758,12 +1760,20 @@ def __AptInstallPackage(self, PackageList=None, ReleaseType=None):
                         
                         if self.ReleaseType is not None:
                                 os.environ['__apt_set_install_release'] = self.ReleaseType
-                                if self.__ExecSystemCmd( '/usr/bin/apt-get -qq --print-uris -t $__apt_set_install_release install $__apt_set_install_packages >> $__apt_set_install' ) is False:
-                                        log.err( "FATAL: Something is wrong with the apt system.\n" )
+                                cmd = '/usr/bin/apt-get -qq --print-uris %s -t $__apt_set_install_release install $__apt_set_install_packages >> $__apt_set_install'
                         else:
                                 #FIXME: Find a more Pythonic implementation
-                                if self.__ExecSystemCmd( '/usr/bin/apt-get -qq --print-uris install $__apt_set_install_packages >> $__apt_set_install' ) is False:
-                                        log.err( "FATAL: Something is wrong with the apt system.\n" )
+                                cmd = '/usr/bin/apt-get -qq --print-uris %s install $__apt_set_install_packages >> $__apt_set_install'
+
+                        reinstall = ''
+                        if self.AptReinstall:
+                                reinstall = '--reinstall'
+
+                        cmd = cmd % reinstall
```

Comment:
No, just string formatting (i've added %s to command line string, i think you have'nt noticed this)
```python
cmd = 'apt-get %s install some_package'
reinstall = ''
cmd = cmd % reinstall
# cmd = 'apt-get   install some_package'
reinstall = '--reinstall'
cmd = cmd % reinstall
# cmd = 'apt-get --reinstall install some_package'
```